src/audio/google/google_rtc_audio_processing.c

```diff
@@ -618,38 +620,14 @@ static int google_rtc_audio_processing_prepare(struct processing_module *mod,
 	return 0;
 }
 
-/* Extracts connected buffer pointers.  Clumsy, should cache in comp_data */
-static void find_bufs(struct processing_module *mod, struct comp_buffer **mic,
```

Comment:
can you collapse this with the patch that added this?

Reply:
Yeah, this is messy.  I'll do a bunch of squashing for the next spin.